scripts/us_bts/latch/constants.py

```diff
@@ -0,0 +1,244 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Constant value used in processing US Tract are defined here.
+This module also consists of mapping between various forms of column names
+found in downloaded files and its corresponding SV name.
+While preprocessing files column names are changed to SV names as used in
+DC import
+"""
+
+INPUT_URLS_CONFIG = "input_urls_config.json"
+
+BASE_URL = "base_url"
+
+FILE_NAMES = "file_names"
+
+DEFAULT_MEASUREMENT_METHOD = "NationalHouseholdTransportationSurveyEstimates"
+
+ACS_SRUVEY_MEASUREMENT_METHOD = DEFAULT_MEASUREMENT_METHOD \
+                                    + "_MarginOfErrorMoreThanACSSurvey"
+
+DOWNLOAD_DIRECTORY = "input_files"
+
+MELT_VAR_COL = "householdSize_numberOfVehicles"
+
+MELT_OBV_COL = "observation"
+
+_HOUSEHOLD_PV = "[dcs:Person {person}]"
+
+_NUM_OF_VEHICLES_PV = "[dcs:AvailableVehicles {vehicle}]"
+
+# pylint: disable=unnecessary-lambda-assignment
+# pylint: disable=line-too-long
+_PV_FORMAT = lambda prop_val: f'"{prop_val[0]}": "dcs:{prop_val[1]}"' if 'None' not in prop_val[
+    1] else ""
```

Comment:
if empty value is also not expected, can change `if 'None' not in prop_val[1]` into `if prop_val[1]`

Reply:
Checking for None string in values.